packages/react/package.json

```diff
@@ -107,7 +106,6 @@
     "normalize.css": "^8.0.1",
     "prop-types": "^15.8.1",
     "react-virtuoso": "^4.3.10",
-    "reactjs-popup": "^2.0.5",
```

Comment:
wasn't this used for the emoji picker?

Reply:
I have used normal Box for that and positioned it correctly.. So no need of this library now ! You can check in the video.. 

Also earlier, it was moving out of the boundary of EC, and was overflowing.. No such issues now !

Reply:
cool